pytest_flask/pytest_compat.py

```diff
@@ -0,0 +1,5 @@
+def getfixturevalue(request, value):
+    if hasattr(request, 'getfixturevalue'):
+        return request.getfixturevalue(value)
+
+    return request(value)
```

Comment:
Nice catch! By accident I overwritten that line in 8aba5e0 before committing my changes and didn't fix it properly in 0c92db3. Sorry.